shunting-yard-replicator/src/main/java/com/hotels/shunting/yard/replicator/exec/event/aggregation/EventMerger.java

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright (C) 2016-2018 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.shunting.yard.replicator.exec.event.aggregation;
+
+import static java.lang.Boolean.TRUE;
+
+import static org.apache.hadoop.hive.common.StatsSetupConst.CASCADE;
+
+import static com.google.common.base.Preconditions.checkArgument;
+
+import static com.hotels.shunting.yard.common.event.EventType.ON_DROP_PARTITION;
+import static com.hotels.shunting.yard.common.event.EventType.ON_DROP_TABLE;
+
+import java.util.Objects;
+
+import com.google.common.collect.ImmutableMap;
+
+import com.hotels.shunting.yard.replicator.exec.event.MetaStoreEvent;
+import com.hotels.shunting.yard.replicator.exec.event.MetaStoreEvent.Builder;
+
+class EventMerger {
+
+  EventMerger() {}
+
+  private boolean isDropEvent(MetaStoreEvent event) {
```

Comment:
Repeated here and in previous class, perhaps this can be refactored?

Reply:
done